lib/utils/scope-util.js

```diff
@@ -0,0 +1,16 @@
+const isFormat = require('@node-oauth/formats');
+const InvalidScopeError = require('../errors/invalid-scope-error');
+
+module.exports = {
+  parseScope: function (requestedScope) {
```

Comment:
Please run the tests with code coverage and if there is missing coverage then please add a unit-test file for this module, testing the parseScope standalone against the cases:

- valid format
  - single word
  - multiple words
  - null
- invalid format
  - single word
  - multiple words

if the other tests already cover this method entirely, please ignore

Reply:
All those cases were previously already covered.

I forgot a single line to cover, so I fixed this by adding one more test!